src/test/java/ProfileTests.java

```diff
@@ -0,0 +1,37 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import Pages.ProfilePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.devtools.v85.profiler.model.Profile;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ProfileTests extends BaseTest {
+    @Test
+    public static void updateProfileNameTest () throws Exception {
```

Comment:
Please, delete `throws Exception`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/219591542-525c329b-3cd4-4913-9d29-b333159c3d62.png)